massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -610,6 +610,7 @@ impl ConsensusWorker {
                     finished = true;
                     break;
                 }
+                self.block_db.storage.store_operation(op);
```

Comment:
Note: I think we can remove this because the operation should have been stored previously by pool.